.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!-- 
+    Thank you for your contribution to the polyPod repo. ♡
+
+    Please, make sure that:
+    - Your code builds clean without any errors or warnings (You've run `./build.js` locally)
```

Comment:
Probably not needed, since it will complain if it does not.

Reply:
Maybe just say wait for the tests to pass, and fix whatever is needed when you possibly can.